### DISCUSSION TITLE
Systemd  = Failed to set up standard output: No such file or directory

### DISCUSSION THREAD

**Discussion by DEPSTRCZ** [pocketbase/pocketbase#1962] [Q&A] [answered]
I have a small problem when trying to use Systemd (from the docs)

When i run this in terminal:
`/PB/PocketBase/pocketbase serve --http="127.0.0.1:8090"` Works just fine...

But when systemd has to run it it fails with:
` systemd[3563163]: pocketbase.service: Failed to set up standard output: No such file or directory
systemd[3563163]: pocketbase.service: Failed at step STDOUT spawning /PB/PocketBase/pocketbase: No such file or directory
systemd[1]: pocketbase.service: Main process exited, code=exited, status=209/STDOUT
systemd[1]: pocketbase.service: Failed with result 'exit-code'.`

Anyone has some advice?

**Comment by ganigeorgiev** [comment, marked as answer]
Are you sure that you've saved the service file in the correct location (the services directory may differ depending on the OS you are using, whether you've specified a systemd mode, etc.)?

---

Actually on second read, the error `Failed at step STDOUT spawning ...` indicates that maybe there is something wrong with the service file (eg. typo, wrong user/group, missing or inaccessible `StandardOutput/StandardError` directory, etc.). Could you share your service file?

**Reply by DEPSTRCZ** [reply]
`[Unit]
Description = pocketbase

[Service]
Type           = simple
User           = root
Group          = root
LimitNOFILE    = 4096
Restart        = always
RestartSec     = 5s
StandardOutput = append:/root/pb/errors.log
StandardError  = append:/root/pb/errors.log
WorkingDirectory = /PB/PocketBase/
ExecStart      = /PB/PocketBase/pocketbase serve --http="127.0.0.1:8090"

[Install]
WantedBy = multi-user.target
`

On the second look i think i forgot to change StandardOutput StandardError

**Reply by ganigeorgiev** [reply]
> On the second look i think i forgot to change StandardOutput StandardError

Yes, I think this is causing the error in your case. Make sure that `/root/pb/` directory exists or change it to `/PB/PocketBase/`.

Additionally, you may not need the `WorkingDirectory` option since you are specifying the absolute path anyway.

**Reply by DEPSTRCZ** [reply]
Thank you for reply, that made me look at it and see it. The problem has been solved..
Caused by StandardOutput and StandardError options having wrong dirs
